Implement stickerResourceType property

clsung opened the pull request:
implments #228

okue commented:
Thanks for your PR.

Would you merge master into your branch? (I modified the pydocstring which causes the CI to fail)

clsung commented:
> Thanks for your PR.
> 
> Would you merge master into your branch? (I modified the pydocstring which causes the CI to fail)

rebased, thanks for notification

okue commented:
Would you please add an item `sticker_resource_type` to the following place?
https://github.com/line/line-bot-sdk-python/blame/b93d92954a66a77a057f3cc60098d1f6740998ee/README.rst#L1059-L1063

cryeo commented on tests/test_webhook.py:
```diff
@@ -452,6 +453,11 @@ def message_content(event):
         def message_sticker(event):
             self.assertEqual('message', event.type)
             self.assertEqual('sticker', event.message.type)
+            self.assertIn(
```
[FYI]
From [documentation](https://developers.line.biz/en/reference/messaging-api/#wh-sticker), sticker resource types may be added without announcing. 

[IMO]
@okue 
This implementation is good enough, but how about the following checking to prevent some confusion?

```py
self.assertEqual('STATIC', event.message. sticker_resource_type)
```

okue replied:
I think `assertIn` is better than `assertEqual`.
Since this method is decorated by `handler.add(MessageEvent, message=StickerMessage)`, thus event is not always `STATIC`.

cryeo replied:
This check is unnecessary since this unit test is for `WebhookHandler`.

okue replied:
I change my mind. https://github.com/line/line-bot-sdk-python/pull/229#discussion_r355872811

clsung commented:
> Would you please add an item `sticker_resource_type` to the following place?
> https://github.com/line/line-bot-sdk-python/blame/b93d92954a66a77a057f3cc60098d1f6740998ee/README.rst#L1059-L1063

Added, thanks.

okue commented on tests/test_webhook.py:
```diff
@@ -452,6 +453,11 @@ def message_content(event):
             self.assertEqual('message', event.type)
             self.assertEqual('sticker', event.message.type)
+            self.assertIn(
+                event.message.sticker_resource_type,
```
This is a test checking if an `event` passed to `message_sticker` method is `StickerMessage` and its message is `StickerMessage`.
So, we should remove an `assertIn` sentence.

clsung replied:
ok
